app/src/ui/chairlift_data.py

```diff
@@ -1,17 +1,16 @@
 from dataclasses import dataclass
 from typing import List
+from snowflake.snowpark import Session
 
 import streamlit as st
-from snowflake.snowpark.context import get_active_session
-session = get_active_session()
 
 @dataclass
 class Machine:
     uuid: str
     name: str
 
 @st.cache_data
-def get_machines() -> List[Machine]:
+def get_machines(session: Session) -> List[Machine]:
```

Comment:
why is it necessary to pass in a session now? It's unused by the implementation

Reply:
session is used on line 14. Before, the module itself tries to get the session, but it is not ideal since it's just a library.

Reply:
oh wow, my bad, resolving